jboss-kie-wildfly-common/added/launch/jboss-kie-wildfly-security-login-modules.sh

```diff
@@ -71,7 +72,11 @@ function configure_ldap_login_module() {
     # RHPAM-1422, if the RealmDirect is set as Required, ldap auth will fail.
     # TODO remove it out as part of the CLOUD-2750
     sed -i 's|<login-module code="RealmDirect" flag="required">|<login-module code="RealmDirect" flag="optional">|' "${CONFIG_FILE}"
-    local login_module=$(build_login_module "LdapExtended" "required")
+    local ldap_login_module_flag="${AUTH_LDAP_LOGIN_MODULE:-required}"
+    if [ "${ldap_login_module_flag}" = "optional" ]; then
+        ldap_login_module_flag="optional"
+    fi
```

Comment:
Is this if needed?

Reply:
@jakubschwan this was the way to validate that we will have only two values, optional or required.